ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-master.yaml

```diff
@@ -22,6 +22,13 @@ images:
       - destination_dir: .
         source_path: /go/src/github.com/integr8ly/integreatly-operator/build
   to: integreatly-operator
+- dockerfile_path: Dockerfile.osde2e
+  from: os
+  inputs:
+    openshift_release_golang-1.13:
+      as:
+      - registry.svc.ci.openshift.org/openshift/release:golang-1.13
+  to: integreatly-operator-test-harness-osde2e
```

Comment:
Looks fine. You will need one of these to approve changes here https://github.com/openshift/release/blob/master/ci-operator/config/integr8ly/integreatly-operator/OWNERS